collector/netframework_clrexceptions.go

```diff
@@ -0,0 +1,132 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRExceptions
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRExceptions class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrexceptions"] = NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector
+}
+
+// A Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRExceptions metrics
+type Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector struct {
+	NumberofExcepsThrown       *prometheus.Desc
+	NumberofExcepsThrownPersec *prometheus.Desc
+	NumberofFiltersPersec      *prometheus.Desc
+	NumberofFinallysPersec     *prometheus.Desc
+	ThrowToCatchDepthPersec    *prometheus.Desc
+}
+
+// NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector ...
+func NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector() (Collector, error) {
+	const subsystem = "win32_perfrawdata_netframework_netclrexceptions"
+	return &Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector{
+		NumberofExcepsThrown: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "numberof_exceps_thrown"),
```

Comment:
Just like the field names, the metric name should have an as clear name as possible, as well as conform to the Prometheus [naming best practices](https://prometheus.io/docs/practices/naming/). So `exceptions_thrown_total` would be one potential name here.